libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -61,6 +61,25 @@ export class StripeManager {
     });
   }
 
+  /**
+   * Cancels incomplete subscription
+   */
+  async cancelIncompleteSubscriptionsToPrice(
+    customerId: string,
+    priceId: string
+  ) {
+    const subscriptions = await this.getSubscriptions(customerId);
+    const targetSubs = subscriptions.data.filter(
+      (sub) => sub.items.data.find((item) => item.price.id === priceId) !== null
```

Comment:
Q: What is `.find() !== null` doing? Just checking that the specified priceId is found in the &lt;whatever&gt;?
Should it use `(sub) => sub.items.data.includes((item) => item.price.id === priceId)` since `.includes()` returns a Boolean and then we don't have to do a nullish check?

Reply:
It could. I may have borrowed legacy code here. :eyes:

Reply:
The nullish check isn't needed regardless of whether `includes` or `find` is used here -- it should be removed